### PR TITLE
Issue/better db flow

### PR DESCRIPTION
**Pull request by kennygperez**

### CODE REVIEW

File: backend/package.json [hunk 1]
```diff
@@ -4,43 +4,39 @@
 	"description": "",
 	"main": "index.js",
 	"scripts": {
-		"build": "tsc",
-		"db:build": "serverless invoke local --function dbInit",
+		"db:init": "serverless invoke local --function dbInit",
```
Comment (Goff-Davis) [inline backend/package.json:7]: IMO there should be a drop and build function for dynamo and mysql as well as one that does both.

**Review by Goff-Davis — CHANGES_REQUESTED** [review]
One thing, otherwise looks good

File: backend/package.json (hunk 1 above)
Comment (kennygperez) [reply backend/package.json:9]: Added redundancy

**Review by ljoks — APPROVED** [review]

**Review by Goff-Davis — APPROVED** [review]
LGTM

**Pull request merged**